src/bam/mod.rs

```diff
@@ -1720,14 +1720,24 @@ CCCCCCCCCCCCCCCCCCC"[..],
                         Writer::from_path(&output_bam_path, &header, Format::BAM).unwrap();
                     writer.set_compression_level(*level).unwrap();
                     for record in reader.records() {
-                        writer.write(&record.unwrap()).unwrap();
+                        let r = record.unwrap();
+                        writer.write(&r).unwrap();
                     }
                 }
                 fs::metadata(output_bam_path).unwrap().len()
             })
             .collect();
 
         // check that out BAM file sizes are in decreasing order, in line with the expected compression factor
+        println!("testing compression leves: {:?}", levels_to_test);
+        println!("got compressed sizes: {:?}", file_sizes);
+
+        // libdeflater comes out with a slightly bigger file on Max compression
```

Comment:
How much bigger if you have the numbers handy? Just curious :)

Reply:
It's something like 450 bytes on Maximum and 438 bytes on Level 6 for this tiny example.